sub/osd_libass.c

```diff
@@ -428,7 +425,7 @@ static void update_progbar(struct osd_state *osd, struct osd_object *obj)
 
     struct ass_draw *d = &(struct ass_draw) { .scale = 4 };
 
-    if (osd->opts->osd_style->back_color.a) {
+    if (osd->opts->osd_style->back_color.a && osd->opts->osd_style->border_style != 1) {
```

Comment:
This makes the back color not work with `BorderStyle=1` at all. Is that the intention? I just assumed the default for `osd-back-color` would be changed to be fully transparent so it could otherwise be configurable.

Reply:
With `BorderStyle=1`, `--osd-back-color` means the shadow color. If users want a background color, they need to set `--osd-border-style=background-box` as well.

Reply:
Sure. Just wanted to make sure that made sense.